### DISCUSSION TITLE
NDS Lite bluetooth

### DISCUSSION THREAD

**Discussion by Gand46** [darthcloud/BlueRetro#127] [Q&A]
I was wondering if its posible connect a bluethooth controller to the nds lite, the reason is for make a DIY gameboy and nds console with the nds tv output  `https://hackaday.com/2021/02/28/hidden-tv-out-on-the-nintendo-ds-lite/`

**Comment by davidxgames** [comment]
this is what you are looking for:
https://youtube.com/shorts/xKjY4GXJV04?feature=share
